kafka/src/main/java/io/specmesh/kafka/provision/schema/SchemaReaders.java

```diff
@@ -203,7 +197,7 @@ private List<ParsedSchema> resolvePayload(final String type, final String conten
 
         private ParsedSchema parsedSchema(final String type, final String payload) {
             if (type.endsWith(".avsc") || type.equals("AVRO")) {
-                return new AvroSchema(payload);
+                return new AvroSchema(payload, List.of(), Map.of(), -1);
```

Comment:
Testing highlighted a bug - when comparing old and new schemas, even a matching Avro schema was reporting itself as changed, as the version was `null`, rather than `-1`.